specification/gp-connect-user-permissions.yaml

```diff
@@ -895,6 +1071,46 @@ components:
       value:
         permissionType: prescriptions
         accessLevel: full
+    InvalidRequestError:
+      value:
+        resourceType: OperationOutcome
+        meta:
+          lastUpdated: "2021-04-14T11:35:00+00:00"
+        issue:
+          - severity: error
+            code: value
+            details:
+              coding:
+                - system: https://fhir.nhs.uk/CodeSystem/Spine-ErrorOrWarningCode
+                  version: "1"
+                  code: INVALID_VALUE
+                  display: Provided value is invalid.
+              diagnostics: (invalid_request) firstName is missing.
+              expression:
+                - /data/attributes/firstName
+    ServerError:
```

Comment:
Is it just the 4XX responses going back in? If so, I think the ServerError example I put in for 5XX errors so we wouldn't need this one?

Reply:
Yes, good call. I thought the lint was picking up unused schemas but it seems to have with this one!